src/db_structures.php

```diff
@@ -0,0 +1,49 @@
+<?php
+//function that returns the database data
+function data(): PDO
+{
+    $db = new PDO(
+        'mysql:host=DB;dbname=collectorapp',
+        'root',
+        'password'
+
+    );
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+//function that returns only the dog array from database
```

Comment:
Remove comments like these from production-ready code. The function getDogData is already telling the reader that the function gets dog-related data, so this sort of comment is redundant.